molecule/default/tests/test_default.py

```diff
@@ -22,22 +22,20 @@ def test_nessus_agent_installed(host):
     assert host.run_expect([0], f'[ -n "$(ls -A {dir_full_path})" ]')
 
 
-def test_nessus_agent_enabled(host):
-    """Test that Nessus Agent is enabled."""
-    assert host.service("nessusagent").is_enabled
+def test_nessus_agent_enabled_and_started(host):
+    """Test that Nessus Agent is enabled and started."""
+    svc = host.service("nessusagent")
+    assert svc.is_enabled and svc.is_running
 
 
 @pytest.mark.parametrize(
     "key, value",
     [
-        ("groups", "good,bad,ugly"),
-        ("key", "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"),
-        ("ms_server_ip", "nessus.example.com"),
-        ("ms_server_port", "8080"),
+        ("update_hostname", "yes"),
     ],
 )
 def test_nessus_config(host, key, value):
-    """Test that Nessus Agent is configured."""
+    """Test that Nessus Agent non-secure configuration options are as expected."""
     assert value in host.check_output(
-        f"/opt/nessus_agent/sbin/nessuscli fix --secure --get {key}"
+        f"/opt/nessus_agent/sbin/nessuscli fix --get {key}"
```

Comment:
Can you let us know why the `--secure` flag was removed here?

Reply:
[The `--secure` flag](https://docs.tenable.com/nessusagent/Content/SettingsAdvanced.htm#SecureSettings) is used to interact with a subset of configuration options that `nessuscli` regards as "secure".  It is not used for interacting with standard AKA non-secure configuration options.

When I was performing the linking configuration in this role (i.e. in the code as it was before this PR) I was checking in the `molecule` tests that a list of four _secure_ configuration options that were part of the linking command were set as expected:  `groups`, `key`, `ms_server_ip`, and `ms_server_port`.  In this pull request I no longer set those four _secure_ options, but I added one _standard_ AKA _non-secure_ option:  `update_hostname`.  Now in the `molecule` tests I check that that one standard option was set as expected; hence, I no longer need the `--secure` flag.